frontend/packages/employee-frontend/src/App.tsx

```diff
@@ -157,28 +141,15 @@ function App() {
               component={ensureAuthenticated(DecisionPage)}
               title={i18n.titles.decision}
             />
-            <RouteWithTitle
-              exact
-              path="/fee-decisions"
-              component={ensureAuthenticated(FeeDecisionsPage)}
-              title={i18n.titles.feeDecisions}
-            />
-            <RouteWithTitle
-              exact
-              path="/fee-decisions/:id"
-              component={ensureAuthenticated(FeeDecisionDetailsPage)}
-            />
-            <RouteWithTitle
+            <Route
               exact
-              path="/invoices"
-              component={ensureAuthenticated(InvoicesPage)}
-              title={i18n.titles.invoices}
+              path="/finance/fee-decisions/:id"
+              component={FeeDecisionDetailsPage}
```

Comment:
Hmm shouldn't we have `ensureAuthenticated` here too?

Reply:
Yeah, good catch.